cachetable.go

```diff
@@ -259,8 +259,8 @@ func (table *CacheTable) Exists(key interface{}) bool {
 	return ok
 }
 
-// NotFoundAdd tests whether an item not found in the cache. Unlike the Exists
-// method this also adds data if they key could not be found.
+// NotFoundAdd checks whether an item not found in the cache. Unlike the Exists
```

Comment:
Careful, this still contains the grammatical error (missing verb). Please change it to the line I suggested. Thank you!

Reply:
Sorry, my mistake